packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -493,7 +493,7 @@ - (void)_createUpdateWithManifest:(NSDictionary *)manifest
 
     if (signatureValidationResult.validationResult == EXUpdatesValidationResultInvalid) {
       NSString *errorMessage = @"Manifest download was successful, but signature was incorrect";
-      [self->_logger error:errorMessage code:EXUpdatesErrorCodeUpdateHasInvalidSignature];
+      [self->_logger error:errorMessage code:EXUpdatesErrorCodeUpdateCodeSigningError];
```

Comment:
Could we not reuse the `HasInvalidSignature` error code instead of creating a new error code?

Reply:
On both android and iOS, that one was used for the legacy signing manifest signing mechanism, so I assumed that the use of it on iOS for the new code signing was a bug (it wasn't applied to the android new code signing). I think they are distinct enough from each other to be considered separate systems and therefore a new code probably makes sense?

Reply:
Let's discuss this a bit tomorrow -- I think you are right, but I want to make sure that the name of the new error code is not confusing to customers :)